src/Symfony/Component/Messenger/Command/SetupTransportsCommand.php

```diff
@@ -81,4 +83,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return 0;
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if (!$input->mustSuggestArgumentValuesFor('transport')) {
+            return;
+        }
+
+        $suggestions->suggestValues($this->transportNames);
```

Comment:
```suggestion
        if (!$input->mustSuggestArgumentValuesFor('transport')) {
            $suggestions->suggestValues($this->transportNames);

            return;
        }
```

While normally, we favor "early return" in the Symfony code base, I propose to not do that in the `complete()` methods. This way, we minimize the diff (and potential merge conflicts) when we add more completion suggestions to this method in a future version.